packages/augur-state/src/db/AbstractDB.ts

```diff
@@ -0,0 +1,69 @@
+import * as PouchDB from "pouchdb";
+import * as _ from "lodash";
+
+interface DocumentIDToRev {
```

Comment:
What does "rev" mean here?

Reply:
This would be for the PouchDB (CouchDB) document revision